### DISCUSSION TITLE
How to connect via LDAP with Thin Client?

### DISCUSSION THREAD

**Discussion by iorabi** [oracle/node-oracledb#1580] [Q&A] [answered]
I am trying to upgrade from v5.5.0 (thick client) to v6.0.2 (thin client), however I currently use `ldap.ora` and `sqlnet.ora` files to connect via LDAP. Is connecting via LDAP compatible with the v6.0.2 thin client?

**Comment by cjbj** [comment, marked as answer]
It is not.  Similar to https://github.com/oracle/python-oracledb/issues/111. You can continue to use Thick mode.

(One buglet I just noticed is that both node-oracledb and python-oracledb docs don't mention it.)

**Reply by sosoba** [reply]
I suggest to add subgroups like "authentication", "data types" etc. in the [feature table](https://node-oracledb.readthedocs.io/en/latest/user_guide/appendix_a.html#id1) and link to it from the main project page.

**Reply by iorabi** [reply]
@cjbj Should I open an enhancement request for this or is it covered by https://github.com/oracle/python-oracledb/issues/111

**Reply by iorabi** [reply]
@cjbj I haven't seen any updates on the python enhancement request. Any idea if this will be addressed in the near future? The thick client is fine for now, but makes development on M1 / M2 less than ideal by requiring different installs of node
